### PR TITLE
Add clarity to register command

### LINKED ISSUE #224: Specify in guides only one user allowed

### PR DESCRIPTION
**Pull request by jianyangg**
Added description to register command
Close #224

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -46,9 +46,13 @@ the instructions in the Getting Started Guide to start using the software.
...
 
-**Format:** `register --user USERNAME --password PASSWORD --confirmPass CONFIRM_PASSWORD --secretQn SECRET_QUESTION --answer ANSWER`
+You can register for an account to use HomeBoss. Only one account can be registered at a time.
 
-**Example:** `register --user gabriel --password gabrielIsGreat --confirmPass gabrielIsGreat --secretQn First pet's name? --answer Koko`
+**Format:
+** `register --user USERNAME --password PASSWORD --confirmPass CONFIRM_PASSWORD --secretQn SECRET_QUESTION --answer ANSWER`
+
+**Example:
+** `register --user gabriel --password gabrielIsGreat --confirmPass gabrielIsGreat --secretQn First pet's name? --answer Koko`
```
Comment (B-enguin) [inline docs/UserGuide.md:55]: Should these newlines be added?

**Review by B-enguin — COMMENTED** [review]
Some Comments

File: docs/UserGuide.md (hunk 1 above)
Comment (jianyangg) [reply docs/UserGuide.md:55]: Auto-save issues with Intelli-J

**Review by juliusgambe — APPROVED** [review]
LGTM

**Pull request merged**